types/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// TypeScript Version: 3.7
+
+import {Node, Parent, Literal} from 'unist'
+
+declare namespace unistUtilAssert {
+  /**
+   * A unist Node that is neither a Parent nor a Literal.
+   */
+  interface VoidNode extends Node {
```

Comment:
@Rokt33r @wooorm thoughts on this name?
I originally made this `VoidNode` instead of `Void` to avoid confusion with the keyword `void`, other suggestions are welcome

Reply:
`VoidNode` sounds good to me.

Reply:
I have a slight preference for `Void`, as it is similar to all the other types: I prefer consistency.

The capitalised version can be freely used in TS, right?

Reply:
It can be, updated to use `Void`